### PR TITLE
feat: tailwind

### PR DESCRIPTION
**Pull request by emersonlaurentino**
## What's the purpose of this pull request?
<em>Considering the context, what is the problem we'll solve? Where in VTEX's big picture our issue fits in? Write a tweet about the context and the problem itself.</em>

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
<em>Describe the steps with bullet points. Is there any external reference, link, or example?</em> 

## References
<em>Spread the knowledge: is this any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

**Comment by igorbrasileiro** [comment]
Did we decide to use a CSS framework here?

File: src/typings/schema.graphql.d.ts [hunk 1]
```diff
@@ -262,6 +262,8 @@ type DirectoryCtimeArgs = {
 type Site = Node & {
   buildTime: Maybe<Scalars['Date']>;
   siteMetadata: Maybe<SiteSiteMetadata>;
+  port: Maybe<Scalars['Int']>;
```
Comment (igorbrasileiro) [inline src/typings/schema.graphql.d.ts:265]: Should this file be here with these changes?

**Review by igorbrasileiro — APPROVED** [review]
LGTM, just take a look at schema.graphql.d.ts file, i don't know if it should have that change.

File: src/typings/schema.graphql.d.ts (hunk 1 above)
Comment (emersonlaurentino) [reply src/typings/schema.graphql.d.ts:265]: nop, for that I opened another PR for it. Can you review it? https://github.com/vtex-sites/base.store/pull/57

This PR will merged after the #57 was merged.

**Review by tlgimenes — APPROVED** [review]
Great!

**Pull request merged**